src/scripts/cleanup.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Script clean up registry and related airflow databases
+ */
+import "dotenv/config";
+import log from "logger";
+import { version } from "utilities";
+import { createDockerJWT } from "registry/jwt";
+import { prisma } from "generated/client";
+import { removeDatabaseForDeployment } from "deployments/database";
+import request from "request-promise-native";
+import config from "config";
+import { DEPLOYMENT_AIRFLOW } from "constants";
+
+const MEDIATYPE_DOCKER_MANIFEST_V2 =
+  "application/vnd.docker.distribution.manifest.v2+json";
+
+async function getManifestoForSelectedTag(dockerJWT, registry, repo, tag) {
+  let uri;
+  uri = `http://${registry}/v2/${repo}/manifests/${tag}`;
+  log.debug(`Requesting docker tag manifest for ${uri} at ${tag}`);
+  let response = await request({
+    method: "GET",
+    uri,
+    json: true,
+    resolveWithFullResponse: true,
+    headers: {
+      Authorization: `Bearer ${dockerJWT}`,
+      Accept: MEDIATYPE_DOCKER_MANIFEST_V2
+    }
+  });
+  return response.headers["docker-content-digest"];
+}
+
+async function deleteManifest(registry, repo, digestHash) {
+  let uri;
+  uri = `http://${registry}/v2/${repo}/manifests/${digestHash}`;
+  log.debug(`deleting ${digestHash}: DELETE ${uri}`);
+  const dockerJWT = createDockerJWT("houston", [
+    {
+      type: "repository",
+      name: repo,
+      actions: ["delete"]
+    }
+  ]);
+  await request({
+    method: "DELETE",
+    uri,
+    json: true,
+    headers: {
+      Authorization: `Bearer ${dockerJWT}`,
+      "User-Agent": `houston/${version()}`
+    }
+  });
+}
+
+async function cleanupRegistry(deployment) {
+  const {
+    releaseNamespace: namespace,
+    releaseName: platformReleaseName
+  } = config.get("helm");
+  const registryPort = config.get("registry.port");
+  const releaseName = deployment.releaseName;
+  const repo = `${releaseName}/${DEPLOYMENT_AIRFLOW}`;
+  const registry = `${platformReleaseName}-registry.${namespace}:${registryPort}`;
+
+  // Create a JWT for the registry request.
+  const dockerJWT = createDockerJWT("houston", [
+    {
+      type: "repository",
+      // Build the repo name.
+      name: repo,
+      actions: ["pull"]
```

Comment:
```suggestion
      actions: ["pull", "delete"]
```

Then we could use the same JWT token for both requests

Reply:
fixed.